lms/djangoapps/gradebook/tests.py

```diff
@@ -34,6 +36,8 @@
 class GradebookTests(ModuleStoreTestCase):
     """ Test suite for Student Gradebook """
 
+    maxDiff = None
```

Comment:
Since this is WIP, I've left this for now. I'll remove this line before merging into master.